x-pack/plugins/reporting/common/cancellation_token.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
nice to see these tests... but is it supposed to catch the issue? I went back to the old code and ran the jest tests and everything still passed. Maybe I'm doing something wrong though.

Reply:
It is a binding issue, I'll see if I can't get a test-case for that specifically since these methods do get passed around

Reply:
Added a new test case, plus changed how some older tests functioned (they did explicit bindings when utilizing this util). Tests started failing when I added these news specs and adjusted older tests.

I'll get it on the list to re-enable that functional test.